### PR TITLE
Fixing file extension to index.html

### PR DESCRIPTION
**Pull request by JoeCamacho**

### CODE REVIEW

**Comment by JoeCamacho** [comment]
Las Commit is done after putting HYFCalculator solution in Joel file

File: Joel/HYFcalculator.html [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+    var resultElement = document.getElementById('js-result');
+    resultElement.innerHTML = result;
+  }
+
+  function add() {
+    
+    num1 = document.getElementById('number1').value;
+    num2 = document.getElementById('number2').value;
+    
+    result = (+num1) + (+num2); /* //My numbers where treated as strings. The solution I found online was to use the unary plus operator,
```
Comment (segersrobbert) [inline Joel/HYFcalculator.html]: Try using the parseInt() javascript function to convert string to number

File: Joel/HYFcalculator.html [hunk 2]
```diff
@@ -0,0 +1,161 @@
...
+    
+    num1 = document.getElementById('number1').value;
+    num2 = document.getElementById('number2').value;
+    
+    result = (+num1) + (+num2); /* //My numbers where treated as strings. The solution I found online was to use the unary plus operator, 
+    but although it works, Im not sure is the best solution */
+    
+    var resultElement = document.getElementById('js-result');
+    resultElement.innerHTML = result;
+    console.log(result);
```
Comment (segersrobbert) [inline Joel/HYFcalculator.html]: remove console.log

File: Joel/HYFcalculator.html [hunk 3]
```diff
@@ -0,0 +1,161 @@
...
+    }
+    var resultElement = document.getElementById('js-result');
+    resultElement.innerHTML = result;
+  }
+
+  function closestTo100() {
+
+    num1 = document.getElementById('number1').value;
+    num2 = document.getElementById('number2').value;
+    let num1Input = Math.abs(Math.abs(num1) - 100);
```
Comment (segersrobbert) [inline Joel/HYFcalculator.html]: Why are you using Math.abs here? Math.abs is to get the absolute value (https://www.w3schools.com/jsref/jsref_abs.asp), not necessary here. The function should be able to take negative numbers also

**Comment by segersrobbert** [comment]
see slack message to avoid global variables num1 and num2

**Pull request merged**